plugins/by-name/lsp-format/default.nix

```
@@ -91,28 +90,40 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     warnings = lib.nixvim.mkWarnings "plugins.lsp-format" {
-      when = !config.plugins.lsp.enable;
+      when = !config.plugins.lspconfig.enable;
```

We currently need to support both `plugins.lsp` and `plugins.lspconfig` side-by side:

```suggestion
      when = !(config.plugins.lsp.enable || config.plugins.lspconfig.enable);
```